agent/php_newrelic.h

```diff
@@ -268,9 +268,14 @@ nrinistr_t browser_monitoring_loader; /* newrelic.browser_monitoring.loader */
 
 nrinibool_t drupal_modules;  /* newrelic.framework.drupal.modules */
 nrinibool_t wordpress_hooks; /* newrelic.framework.wordpress.hooks */
-nrinitime_t wordpress_hooks_threshold; /* newrelic.framework.wordpress.hooks_threshold */
-nrinibool_t wordpress_plugins; /* newrelic.framework.wordpress.plugins */
-nrinibool_t wordpress_core;  /* newrelic.framework.wordpress.core */
+nrinistr_t
+    wordpress_hooks_options; /* newrelic.framework.wordpress.hooks.options */
+nrinitime_t wordpress_hooks_threshold; /* newrelic.framework.wordpress.hooks.threshold
+                                        */
+nrinibool_t wordpress_plugins;         /* set based on
```

Comment:
Since this isn't an INI value, I wonder if this is the best type for it. For example, this value `bool check_cufa;` is an already existing example (in this file) of a value that is dependant on and closely related to an INI value, but does not have the `nrini` type.

Reply:
@ZNeumann Agreed. See 9792e761cc7f558cb525e73adc15add5b814994c.